lib/oauth2-client.js

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import database from 'bedrock-mongodb';
+import * as bedrock from 'bedrock';
+import {promisify} from 'util';
+import {httpClient} from '@digitalbazaar/http-client';
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock-mongodb.ready', async () => {
+  await promisify(database.openCollections)(['access-tokens']);
```

Comment:
We don't need `promisify` here anymore.

```suggestion
  await database.openCollections(['access-tokens']);
```